### PR TITLE
Add Syntax Highlighting Test Files for TeX, PGP, Pascal and 17 other languages

### PR DESCRIPTION
**Pull request by KennethSweezy**
The complete list of languages is:
TeX, SparQL, Stylus, Soy, Smalltalk, Slim, Sieve, Scheme, reStructuredText, RPM, Q, Puppet, Pug, ProtoBuf, Properties, Apache Pig, PGP, Octave, Oz, and Pascal

[Issue](https://github.com/desktop/desktop/issues/7217)

### CODE REVIEW

File: jade.jade [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# == Class: automysqlbackup
```
Comment (niik) [inline jade.jade]: This looks like a puppet file to me, and given that it's the same as the `puppet.pp` is it possible this was a mistaken copy paste?

Comment (KennethSweezy) [reply jade.jade]: Ah yes, it appears that it is. I will fix it.

Comment (KennethSweezy) [reply jade.jade]: Ok, it has been removed now.

File: protobuf.proto [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+// Our address book file is just one of these.
+message AddressBook {
+  repeated Person people = 1;
+}
+service Test {
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
+}
```
Comment (iAmWillShepherd) [inline protobuf.proto]: This file is missing a final newline.

File: octave.m [hunk 3]
```diff
@@ -0,0 +1,42 @@
+%numbers
```
Comment (say25) [inline octave.m]: I believe we dropped octave.m support we should drop this file

Comment (KennethSweezy) [reply octave.m]: Thanks for reminding me about that! It has now been removed.

**Pull request merged**